src/main/java/org/fungover/breeze/Matrix/Matrix3x3.java

```diff
@@ -0,0 +1,157 @@
+package org.fungover.breeze.Matrix;
```

Comment:
Package names should be all lowercase for best compatibility between different operating systems file system.